app/src/main/java/com/codesoom/assignment/models/Task.java

```diff
@@ -0,0 +1,54 @@
+package com.codesoom.assignment.models;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.ToString;
+
+@Getter
+@ToString
+@Builder
+public class Task {
+
+    private Long id;
+    private String title;
+
+    public Task() {
+    }
+
+    public Task(Long id, String title) {
+        this.id = id;
+        this.title = title;
+    }
+
+    public void updateTitle(String title) {
+        this.title = title;
+    }
```

Comment:
이건 이름에만 `set`이 아닌 setter입니다. 우리가 무엇을 목표로 setter 를 사용하지 않기로 하게 된 것인지를 생각해 보셔야 해요.
목표가 불변 객체이므로 다음과 같이 작업하는 것을 고려해보세요.

```suggestion
    public Task updateTitle(String title) {
        return new Title(this.id title);
    }
```

물론 메소드 이름은 더 나은 이름이 가능합니다. 이름에 대해서도 고민해 보세요.
그리고 이렇게 하게 되면 어떤 장단점이 있는지에 대해서도 생각해 보세요.